pkg/workers/exec/konnector.go

```diff
@@ -218,7 +236,59 @@ func (w *konnectorWorker) Error(i *instance.Instance, err error) error {
 }
 
 func (w *konnectorWorker) Commit(ctx *jobs.WorkerContext, errjob error) error {
-	return nil
+	// TODO: remove this retro-compatibility block
+	// <<<<<<<<<<<<<
```

Comment:
This commit puts back the `konnectorResult` logic in order to keep compatibility with current `cozy-collect` and ease its update (cc @CPatchane).

Reply:
Thanks 👍 (poke @gregorylegarec )